ci/scenario_helpers/ci_constants.py

```diff
@@ -0,0 +1,124 @@
+
+    REQUIRED_PACKAGES_HYPERVISOR = ['qemu-kvm', 'libvirt0', 'python-libvirt', 'virtinst']
+    REQUIRED_PACKAGE_CLOUD_INIT = ['genisoimage']
+
+    with open(CONFIG_LOC, 'r') as JSON_CONFIG:
+        SETUP_CFG = json.load(JSON_CONFIG)
+
+    with open(SETTINGS_LOC, 'r') as JSON_SETTINGS:
+        SETTINGS = json.load(JSON_SETTINGS)
+
+    DATA_TEST_CASES = [(0, 100), (30, 70), (40, 60), (50, 50), (70, 30), (100, 0)]  # read write patterns to test (read, write)
+
+    CLOUD_INIT_DATA = {
+        'script_loc': 'https://raw.githubusercontent.com/kinvaris/cloud-init/master/create-config-drive',
+        'script_dest': '/tmp/cloud_init_script.sh',
+        'user-data_loc': '/tmp/user-data-migrate-test',
+        'config_dest': '/tmp/cloud-init-config-migrate-test'
+    }
+
+    # collect details about parent hypervisor
+    PARENT_HYPERVISOR_INFO = SETUP_CFG['ci'].get('hypervisor')
+
+    # hypervisor details
+    HYPERVISOR_TYPE = SETUP_CFG['ci']['local_hypervisor']['type']
+    HYPERVISOR_USER = SETUP_CFG['ci']['local_hypervisor']['user']
+    HYPERVISOR_PASSWORD = SETUP_CFG['ci']['local_hypervisor']['password']
+
+    HYPERVISOR_INFO = {'type': HYPERVISOR_TYPE,
+                       'user': HYPERVISOR_USER,
+                       'password': HYPERVISOR_PASSWORD}
+
+    VM_USERNAME = 'root'  # vm credentials & details
+    VM_PASSWORD = 'rooter'
+    VM_VCPUS = 4
+    VM_VRAM = 1024  # In MB
+    VM_OS_TYPE = 'ubuntu16.04'
+
+    VM_WAIT_TIME = 300  # wait time before timing out on the vm install in seconds
+
+    VDISK_THREAD_LIMIT = 5  # Each monitor thread queries x amount of vdisks
+    FIO_VDISK_LIMIT = 50  # Each fio uses x disks
+
+    IO_REFRESH_RATE = 5  # Refresh rate used for polling IO
+    AMOUNT_TO_WRITE = 1 * 1024 ** 3  # Amount of data to RW to produce IO
+
+    HA_TIMEOUT = 300
+
+    @classmethod
+    def get_api_instance(cls):
+        """
+        Fetches the api instance using the constants provided by the configuration files
+        :return: ovsclient instance
+        :rtype: ci.api_lib.helpers.api.OVSClient
+        """
+        return OVSClient(cls.SETUP_CFG['ci']['grid_ip'],
+                         cls.SETUP_CFG['ci']['user']['api']['username'],
+                         cls.SETUP_CFG['ci']['user']['api']['password'])
+
+    @classmethod
+    def get_parent_hypervisor_instance(cls):
+        """
+        Fetches the parent hypervisor instance
+        :return: 
+        """
+        return HypervisorFactory.get(cls.PARENT_HYPERVISOR_INFO['ip'],
+                                     cls.PARENT_HYPERVISOR_INFO['user'],
+                                     cls.PARENT_HYPERVISOR_INFO['password'],
+                                     cls.PARENT_HYPERVISOR_INFO['type'])
+
+    @classmethod
+    def get_storagerouters_for_ha(cls):
+        """
+        Gets storagerouters based on roles
+        :return: 
+        """
+        voldr_str_1 = None  # Will act as volumedriver node
```

Comment:
This will actually select them from the pool of storagerouters configured in the setup.json.
Based on its role there, a storagerouter will be selected.
Eg: 
```
        "10.100.199.191": {"name": "ubuntu16.04-setup3-ovsnode01", "role": "NONE"},
        "10.100.199.192": {"name": "ubuntu16.04-setup3-ovsnode02", "role": "VOLDRV"},
        "10.100.199.193": {"name": "ubuntu16.04-setup3-ovsnode03", "role": "VOLDRV"},
        "10.100.199.194": {"name": "ubuntu16.04-setup3-ovsnode04", "role": "COMPUTE"}
```